ts/src/bitso.ts

```diff
@@ -1722,7 +1714,7 @@ export default class bitso extends Exchange {
             'txid': this.safeString (details, 'tx_hash'),
             'timestamp': this.parse8601 (datetime),
             'datetime': datetime,
-            'network': this.safeNetwork (networkId),
+            'network': this.networkIdToCode (networkId).toUpperCase (),
```

Comment:
same thing here, what if this is undefined?

Reply:
anyway fixed & pushed, but still lmk your answer on above